### PR TITLE
Loadbalancer Samples Improvements

### PR DESCRIPTION
**Pull request by predic8**
## Summary by CodeRabbit

- **New Features**
  - Added a Health Monitor load‑balancer example (HTTP + TLS) with run/verification steps and failure/recovery scenarios.

- **Documentation**
  - Rewrote Load Balancing READMEs for clarity and cross‑platform "What You Will Run" workflows; added TLS guidance; removed an older TLS README.

- **Examples**
  - Standardized certificate locations and TLS ports/endpoints (LB port 8443, admin 9443); clarified observable alternating responses and health‑driven failover.

- **Tests**
  - Added integration tests for the health‑monitor example; removed/updated some legacy tests and adjusted test visibilities.

- **Chores**
  - Updated .gitattributes for UTF‑16 and PKCS#12 handling; assorted formatting/import cleanups.

### CODE REVIEW

File: core/src/main/java/com/predic8/membrane/core/interceptor/balancer/BalancerHealthMonitor.java [hunk 1]
```diff
@@ -96,8 +105,12 @@ private void updateStatus(Node node) {
     }
 
     private ScheduledExecutorService createScheduler() {
-        ScheduledExecutorService s = Executors.newSingleThreadScheduledExecutor();
-        s.scheduleAtFixedRate(healthCheckTask, interval, interval, MILLISECONDS);
+        ScheduledExecutorService s = newSingleThreadScheduledExecutor(r -> {
```
Comment (rrayst) [inline core/src/main/java/com/predic8/membrane/core/interceptor/balancer/BalancerHealthMonitor.java]: könnte man `router.getTimerManager().schedule(...)` nutzen? Dann wird kein Thread blockiert.

File: core/src/main/java/com/predic8/membrane/core/interceptor/balancer/BalancerHealthMonitor.java [hunk 2]
```diff
@@ -131,7 +144,12 @@ private static String getNodeHealthEndpoint(Node node) {
 
     private Exchange doCall(String url) throws Exception {
         Exchange exc = get(url).buildExchange();
-        client.call(exc);
+        try {
+            client.call(exc);
+        } finally {
+            exc.finishExchange(false,null);
+            exc.detach();
```
Comment (rrayst) [inline core/src/main/java/com/predic8/membrane/core/interceptor/balancer/BalancerHealthMonitor.java]: weder `.finishExchange(...)` noch `.detach()` sind notwendig.

File: distribution/examples/loadbalancing/6-health-monitor/certificates/backend-1.pem [hunk 3]
```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```
Comment (rrayst) [inline distribution/examples/loadbalancing/6-health-monitor/certificates/backend-1.pem:1]: Please add the certificate's SHA256 [here](https://github.com/membrane/api-gateway/blob/e578d355cfde0feb0e22e16c2aff1cae74c14951/core/src/main/java/com/predic8/membrane/core/transport/ssl/StaticSSLContext.java#L311) to issue a warning when it's being used.

File: distribution/examples/loadbalancing/6-health-monitor/certificates/backend-2.pem [hunk 4]
```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```
Comment (rrayst) [inline distribution/examples/loadbalancing/6-health-monitor/certificates/backend-2.pem:1]: Please add the certificate's SHA256 [here](https://github.com/membrane/api-gateway/blob/e578d355cfde0feb0e22e16c2aff1cae74c14951/core/src/main/java/com/predic8/membrane/core/transport/ssl/StaticSSLContext.java#L311) to issue a warning when it's being used.
